src/assets/languages/english.json

```diff
@@ -89,6 +89,19 @@
         "error": "First Name is required"
       }
     },
+    "certificate": {
+      "header": "CERTIFICATE",
+      "congratsMessage": {
+        "messagePart1": "Peace Corps congratulates ",
+        "messagePart2": "for completing all the activities in the ",
+        "brandPart1": "PC Prep",
+        "brandPart2": "Kit"
+      },
+      "infokitMessage": "You may see your earned badges in InfoKit.",
+      "chatbotMessage": "For future, We recommend you for using the ChatBot when in doubts related to Malaria.",
```

Comment:
Suggestion to wording change: In future, we recommend using the ChatBot when in doubts related to Malaria
@Raxerz @shreyans29 @paavininanda please confirm.

Reply:
Yea sounds good @Buddhiprabha

Reply:
Has this been changed? @lunayach